.docker/ubuntu-20.04/Dockerfile

```diff
@@ -39,6 +40,17 @@ RUN apt-get update && apt-get install --no-install-recommends -y apt-transport-h
     cppcheck \
     clang-tidy \
     gdb \
+    libxdp-dev \
```

Comment:
You mentioned in our teams call today that some packages were old versions. Do these have the versions you need?

Reply:
`apt-add-repository "deb http://mirrors.kernel.org/ubuntu noble main"`
I found this repository having 1.4.0. works on Ubuntu 22.04

Reply:
by default there is no libxdp package at all. there is only libbpf 0.5 or so (ubuntu 22.04)

Reply:
xdp-tools seems to be not providing binary.
https://github.com/xdp-project/xdp-tools/issues/416